pyglossary/plugins/ebook_mobi.py

```diff
@@ -69,17 +70,10 @@
 ]
 
 optionsProp = {
-	"group_by_prefix_length": IntOption(
-		comment="Prefix length for grouping",
```

Comment:
Add this back and make is `disabled=True`, because it inherits from ebook_base.py

Reply:
Could I ask for more explain?

The optionsProp dict is not in a inherited class.
 `Add this back and make is disabled=True  ` is not override and set disabled=True to any Existing Option, right？
It just make a dummy Option.

So is it just for a layout style purpose or something?